00-HTML-CSS-basics/HTML/Excersise1/tableOfExpenses.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
The file name it's OK, but as a good practice I think it's better to use `table-of-expenses.html` as example